src/acvp.c

```diff
@@ -1499,6 +1599,17 @@ static ACVP_RESULT acvp_build_register(ACVP_CTX *ctx, char **reg)
             case ACVP_CTRDRBG:
                 acvp_build_drbg_register_cap(cap_obj, cap_entry);
                 break;
+            case ACVP_HMAC_SHA1:
+            case ACVP_HMAC_SHA2_224:
+            case ACVP_HMAC_SHA2_256:
+            case ACVP_HMAC_SHA2_384:
+            case ACVP_HMAC_SHA2_512:
+            case ACVP_HMAC_SHA3_224:
```

Comment:
Same comment here - for HMAC-SHA3 we may want an unsupported algorithm return code here